src/Tests/Topology/When_counting_existing_topics_in_bundle.cs

```diff
@@ -0,0 +1,53 @@
+﻿namespace NServiceBus.Azure.WindowsAzureServiceBus.Tests.Topology
+{
+    using System;
+    using System.Text.RegularExpressions;
+    using System.Threading.Tasks;
+    using Microsoft.ServiceBus;
+    using Microsoft.ServiceBus.Messaging;
+    using NUnit.Framework;
+    using TestUtils;
+    using Transport.AzureServiceBus;
+
+    [TestFixture]
+    [Category("AzureServiceBus")]
+    public class When_counting_existing_topics_in_bundle
+    {
+#pragma warning disable 618
```

Comment:
Pragmas could be removed if we'd go directly to the namespace manager without using our adapter.

Reply:
> This change is not strictly necessary but I guess it is still good

😃

Reply:
👍